### DISCUSSION TITLE
Test report: usability

### DISCUSSION THREAD

**Discussion by nikitavlaev** [UnitTestBot/UTBotJava#453] [General]
Currently, test reports looks like this:
![image](https://user-images.githubusercontent.com/43010567/177751114-5cc1b2dc-de36-4f47-82f1-fb457500cefa.png)
expanded view of the same:
![image](https://user-images.githubusercontent.com/43010567/177751252-b29dbb40-dfda-408c-ada6-ef929ab6ab03.png)
Event log:
![image](https://user-images.githubusercontent.com/43010567/177751294-9e5b3d4b-ac78-4b9d-9388-91075d40fb0c.png)

If there were no test generated, plugin shows this error message:
![image](https://user-images.githubusercontent.com/43010567/177752442-08314c35-8a32-458c-8a39-8588cc653226.png)

There are several problems with that:
1. 2 bubbles -- no good.
2. Bubble with test report is too small -- no warnings can be seen, even expanded balloon requires scroll a lot.
3. SARIF report takes too much place and is of questionable use currently.

The proposed solution is:
1. Show full info only in event log, SARIF appears only there
2. No warnings -- show notification bubble with short report like:

> **Report of the unit tests generation via UtBot**
> Unit tests for class MyClass were generated successfully.
> Overall test methods: 11
> [See details in event log](https://github.com/UnitTestBot/UTBotJava/discussions)

3. There are warnings -- show warning(!) bubble with short report like:
> **Report of the unit tests generation via UtBot**
> Unit tests for class MyClass were generated with warnings.
> Warning: Some test cases were ignored, because no mocking framework is installed in the project. 
> Better results could be achieved by [installing mocking framework](https://github.com/UnitTestBot/UTBotJava/discussions).
> Overall test methods: 11
> [See details in event log](https://github.com/UnitTestBot/UTBotJava/discussions)

**Comment by korifey** [comment]
Text about VSCode extension in IntellijIDEA doesn't seems right to me. Maybe temporary we can give link to Intellij SARIF plugin.

**Comment by mmvpm** [comment]
Is it necessary to display this amount of information?

![image](https://user-images.githubusercontent.com/54814796/177790062-50ad8838-aa07-4455-a6e7-1250e1798ec3.png)
 
I mean, is the information about "how much tests are: successful / failing because of (some reason) / anything else" relevant to the user?

**Reply by Vassiliy-Kudryashov** [reply]
@nikitavlaev suggested **"Overall test methods: 11"** and it looks like the best summary message for me.

**Reply by amandelpie** [reply]
Looks like this is a useful information for UTBot developer in debug mode

**Comment by nikitavlaev** [comment]
Also, tsv is not required as far as I can see.
